src/NServiceBus.Core/Transports/Msmq/InstanceMapping/InstanceMappingTable.cs

```diff
@@ -35,85 +34,22 @@ public void ReloadData()
             {
                 var doc = fileAccess.Load(filePath);
                 var instances = parser.Parse(doc);
-
-                var newInstanceMap = instances
-                    .GroupBy(i => i.Endpoint)
-                    .ToDictionary(g => g.Key, g => new HashSet<EndpointInstance>(g));
-
-                LogChanges(instanceMap, newInstanceMap);
-
-                instanceMap = newInstanceMap;
+                endpointInstances.AddOrReplaceInstances("FileRoutingTable", instances.ToList());
```

Comment:
Should this be "InstanceMappingTable" to match the new naming used for all the other instance-mapping related things?